src/components/Swap/SwapButton/index.tsx

```diff
@@ -67,80 +67,82 @@ export default memo(function SwapButton({ disabled }: SwapButtonProps) {
   // Close the review modal on chain change.
   useEffect(() => setOpen(false), [chainId])
 
-  const addTransaction = useAddTransaction()
+  const addTransactionInfo = useAddTransaction()
```

Comment:
should this hook name change?

Reply:
This is adding a `TransactionInfo` type (`Transaction` type is different), so I'm going to propagate this change to all callsites.